backup/predata_relations.go

```diff
@@ -47,15 +47,21 @@ func SplitTablesByPartitionType(tables []Table, includeList []options.Relation)
 					// and does not need the suffix added
 					table.Name = AppendExtPartSuffix(table.Name)
 				}
-				metadataTables = append(metadataTables, table)
+				if table.ExtensionTableConfig == nil {
```

Comment:
Could you please add comments describing the purpose of this condition?

Reply:
added